app/services/admin/onboarding_wizard.rb

```diff
@@ -103,7 +103,7 @@ def paypal_preferences_updated(setup_status, community)
 
     def listing_created(setup_status, listing)
       if !setup_status[:listing] &&
-         listing
+         listing&.author&.is_marketplace_admin?
```

Comment:
👍 

This now affects the bootstrap logic a bit. The way it works now is that it pulls the first listing for the community and checks that. But since we no longer accept any listing to mean :listing step is complete we should now pull all listings in the bootstrap phase (in batches) and process them through this callback as long as we either get a hit or we run out of listings.